apps/api/src/migration/1729889576502-AddEndOfJourneyApplicantStatus.ts

```diff
@@ -0,0 +1,41 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddEndOfJourneyApplicantStatus1729889576502 implements MigrationInterface {
+  milestones = [
```

Comment:
We'll need to update our documentation to include these IDs in our list of milestones.

Reply:
Added to Wiki:
https://github.com/bcgov/internationally-educated-nurses/wiki/Documentation